napari/layers/base/base.py

```diff
@@ -271,8 +279,12 @@ def blending(self):
 
     @blending.setter
     def blending(self, blending):
-        self._blending = Blending(blending)
-        self.events.blending()
+        self.events.blending(value=blending)
+
+    def _on_blending_change(self, blending):
+        if isinstance(blending, str):
```

Comment:
this is a reversion of #1027.  no longer need to check `isinstance(str)` ... just call `Blending()`

Reply:
noted